spec/classes/rabbitmq_spec.rb

```diff
@@ -41,37 +41,35 @@
         case facts[:osfamily]
         when 'Debian'
           it 'includes rabbitmq::repo::apt' do
-            is_expected.to contain_class('rabbitmq::repo::apt')
-              .with_key_source('https://www.rabbitmq.com/rabbitmq-release-signing-key.asc')
-              .with_key_content(nil)
+            is_expected.to contain_class('rabbitmq::repo::apt').
+              with_key_source('https://www.rabbitmq.com/rabbitmq-release-signing-key.asc').
+              with_key_content(nil)
           end
 
           it 'adds a repo with default values' do
-            is_expected.to contain_apt__source('rabbitmq')
-              .with_ensure('present')
-              .with_location('http://www.rabbitmq.com/debian/')
-              .with_release('testing')
-              .with_repos('main')
+            is_expected.to contain_apt__source('rabbitmq').
+              with_ensure('present').
+              with_location('http://www.rabbitmq.com/debian/').
+              with_release('testing').
+              with_repos('main')
           end
 
-          it { is_expected.not_to contain_class('rabbitmq::repo::rhel') }
```

Comment:
This feels like a false positive to me.

Reply:
Yeah, I'm wondering if it should just be reordered or have slightly different logic?
Maybe it could just be inside a `when not 'xyz'` block?

Reply:
See how the updated version I'm about to push looks (oddly, it needs to be `when !('xyz')`)

Reply:
Not pretty but if it works ...